ui/src/scss/base.scss

```diff
@@ -47,3 +47,25 @@ body {
 .u-upper-case--first:first-letter {
   text-transform: capitalize;
 }
+
+  //XXX can be removed / replaced with whatever mixin will contain the placeholders %vf-list, %vf-list-item used below, once  this is fixed: https://github.com/canonical-web-and-design/vanilla-framework/issues/2808
```

Comment:
Could you put this in the vanilla_overrides file? Just so all the removable stuff is in the same file

Reply:
done